src/silverlabnwb/nwb_file.py

```diff
@@ -775,13 +798,11 @@ def read_zplane(self, zplane_path):
                 name=name,
                 description='Reference Z stack',
                 manifold=manifold)
-        # TODO Define and use an extension to store these
-        # self.nwb_file.set_custom_dataset(
-        #     '/general/optophysiology/zplane_pockels',
-        #     zplane_data.values,
-        #     attrs={'columns': zplane_data.columns.tolist()})
-        # self.nwb_file.set_custom_dataset(
-        #     '/general/optophysiology/frame_size', [num_pixels, num_pixels])
+        ZplanePockelsDatasetClass = get_class('ZplanePockelsDataset', 'silverlab_extended_schema')
+        self.custom_silverlab_dict['zplane_pockels'] = ZplanePockelsDatasetClass(
+            columns=['z', 'znorm', 'laser_power', 'z_motor'],
```

Comment:
I slightly prefer the original version of `zplane_data.columns.tolist()` in case the format changes in the future. I do like that this is clearer about what data there is, but the column names are also mentioned where the `zplane_data` is read a few lines earlier.